### PR TITLE
Fix confusing variable name in "metalearning with GPs" notebook.

### PR DESCRIPTION
**Pull request by christabella**
The return value from `optimization_step` is not really the `elbo` since the model is `GPR` so `model.log_marginal_likelihood()` is exact.

Instead of ELBO, the GPR's loss function is the negative log marginal likelihood (LML).

---

By the way, I have another question about [this "metalearning with GPs" notebook](https://gpflow.readthedocs.io/en/master/notebooks/tailor/external-mean-function.html)!

Is `num_datapoints` both the number of training points $\tilde{n}$ and meta tasks' full data points $n_i$ in the [original paper by Fortuin et al.](https://arxiv.org/abs/1901.08098), i.e. $n_i == \tilde{n}$?
| Notebook code      | Paper notation | Meaning                         |
|--------------------|----------------|---------------------------------|
| `num_datapoints = 5` | n_i            | Meta tasks’ # data points       |
| `num_datapoints = 5` | \tilde{n}      | Target task’s # training points |
| `N = 1000`           | \tilde{n}*     | Target task’s # test points     |

I was just trying to compare it to the setup of Garnelo et al.'s "Neural Processes" where n_i = \tilde{n} + \tilde{n}* > \tilde{n}

| NP [e.g. from Le et al.](http://bayesiandeeplearning.org/2018/papers/92.pdf) | Deep mean GP [from Fortuin et al.](https://arxiv.org/abs/1901.08098) |
|--|--|
| {xC, yC, xT, yT}, the context and target sets for training | meta-task datasets {D_i} for i = 1,...,m |
| For every meta-task, optimize the ELBO to the log predictive likelihood: log p(yT\|xT,xC,yC) | For every meta-task, optimize the log marginal likelihood: log p(yi\|xi) |
| {xC*, yC*}, context set for testing | target task training data { \tilde{x}, \tilde{y} } |
| {xT*, yT*}, target set for testing  | target task test data { \tilde{x}\*, \tilde{y}* } |

{xC*, yC*, xT*, yT*} are not described in Le et al. because they only evaluate log likelihoods and do not predict on unseen tasks (like [the official ANP notebook](https://github.com/deepmind/neural-processes/blob/master/attentive_neural_process.ipynb)).

I am assuming {xC*, yC*, xT*, yT*} are new context sets that are disjoint from {xC, yC, xT, yT} and drawn from entirely new functions/"tasks" at test time.

### CODE REVIEW

**Comment by christabella** [comment]
From email correspondence with Vincent Fortuin, the author of the reference paper for the notebook, I have found out that `n_i` should actually be 50 and `N=50` :) I will create a new PR to update the notebook.

**Comment by christabella** [comment]
Just realized that maybe I should be editing the source file (instead of this Jupyter notebook): https://github.com/GPflow/GPflow/blob/develop/doc/source/notebooks/tailor/external-mean-function.pct.py

**Comment by vdutor** [comment]
@christabella thank you for your PR.  I've seen you made the change to the [source file of the notebook](https://github.com/GPflow/GPflow/blob/develop/doc/source/notebooks/tailor/external-mean-function.pct.py) so I'll close this one.
